Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -367,6 +379,21 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         if (this.delegate && typeof this.delegate.didAppendConsoleMessageView === "function")
             this.delegate.didAppendConsoleMessageView(messageView);
     }
+
+    _handleShowConsoleMessageTimestampsSettingChanged()
+    {
+        this._showTimestamps = WI.settings.showConsoleMessageTimestamps.value;
+        if (this._showTimestamps) {
+            this._currentSessionOrGroup.element.removeAttribute("hide-timestamps");
```

Comment:
Ditto :340-344 (e.g. move this above the if statement and then you can get rid of the else below as well)